.github/workflows/lint.yml

```diff
@@ -0,0 +1,31 @@
+name: Lint
+
+on:
+  push:
+    branches-ignore:
+      - "dependabot/**"
+  pull_request:
+
+env:
+  FORCE_COLOR: 2
+  NODE: 14
```

Comment:
I went with Node.js 14 because it seems npm 6.x is used right now on master. If you switch to npm >= 7.x, you should change this to Node.js 16.